web/src/views/ConsentPortal/OpenIDConnectConsentPortal/OpenIDConnectConsentDecisionFormView.tsx

```
@@ -165,128 +167,143 @@ const OpenIDConnectConsentDecisionFormView: React.FC<Props> = (props: Props) =>
-                                    onClick={handleRejectConsent}
-                                    color="secondary"
-                                    variant="contained"
-                                >
-                                    {translate("Deny")}
-                                </Button>
+                            {hasClaims ? (
+                                <Grid size={{ xs: 12 }}>
+                                    <Box className={classes.claimsListContainer}>
+                                        <List className={classes.claimsList}>
+                                            {response?.essential_claims?.map((claim: string) => (
+                                                <Tooltip title={translate("Claim", { name: claim, ns: "consent" })}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)